Cargo.toml

```diff
@@ -22,6 +23,11 @@ string-interner = { version = "0.14.0", default-features = false, features = ["s
 take_mut = "0.2.2"
 try-from-discrim = "1.0.0"
 warnings-gen = { path = "warnings-gen" }
+nix = "0.28.0"
```

Comment:
This doesn't seem to be used?

Reply:
Good eye, it was leftover when I decided to not use this library anymore. Removed.